src/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -60,7 +61,14 @@ public static IServiceCollection AddOmexServiceFabricDependencies<TContext>(this
 
 			collection.TryAddSingleton<IServiceContext, OmexServiceFabricContext>();
 			collection.TryAddSingleton<IExecutionContext, ServiceFabricExecutionContext>();
+
+			collection.AddLogging();
+
+			collection.TryAddTransient<IExecutionContext, BaseExecutionContext>();
+
+#pragma warning disable OMEX188 // This method also adds Legacy OmexLogger and ActivityEventSender which are deprecated. They are pending for removal by 1 July 2024. Consider adding a different telemetry solution. DiagnosticId = "OMEX188"
 			return collection.AddOmexServices();
+#pragma warning restore OMEX188 // This method also adds Legacy OmexLogger and ActivityEventSender which are deprecated. They are pending for removal by 1 July 2024. Consider adding a different telemetry solution. DiagnosticId = "OMEX188"
```

Comment:
As explained in another comment, these changes are not needed, revert. The line ```collection.TryAddTransient<IExecutionContext, BaseExecutionContext>();``` is not doing anything anyway, it will be simply ignored.

Reply:
done